messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/jms/ExternalConnectionFactoryAdd.java

```diff
@@ -76,7 +76,7 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
         final String name = context.getCurrentAddressValue();
         final ServiceName serviceName = ExternalConnectionFactoryDefinition.CAPABILITY.getCapabilityServiceName(context.getCurrentAddress());
         boolean ha = HA.resolveModelAttribute(context, model).asBoolean();
-        boolean enable1Prefixes = ENABLE_AMQ1_PREFIX.resolveModelAttribute(context, model).asBoolean();
+        boolean enable1Prefixes = ENABLE_AMQ1_PREFIX.resolveModelAttribute(context, model).asBoolean(true);
```

Comment:
This is not needed as the AttributeDefinition has default value ModelNode.TRUE.

Reply:
sorry, leftover